src/app/dashboard/helper.ts

```diff
@@ -0,0 +1,71 @@
+import { Redis } from "@upstash/redis"
+
+// Initialize Redis outside of the job function
+const redis = new Redis({
+  url: process.env.REDIS_URL,
+  token: process.env.REDIS_TOKEN
+})
+
+interface InviteCodeReturnJsonSuccess {
+  isInviteCodeGenerated: boolean
+  inviteCode: string
+}
+
+type InviteCodeReturnJsonFailure = Omit<InviteCodeReturnJsonSuccess, "inviteCode">
+
+export async function generateInviteCode(
+  role: string
+): Promise<InviteCodeReturnJsonSuccess | InviteCodeReturnJsonFailure> {
+  const rString = "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+  let inviteCode = ""
+  for (var i = 0; i < 5; i++) {
```

Comment:
use let instead of var.
```suggestion
  for (let i = 0; i < 5; i++) {
```

Reply:
yeah got it.